spring-cloud-dataflow-server-cloudfoundry/pom.xml

```diff
@@ -13,6 +13,7 @@
 	</parent>
 	<properties>
 		<start-class>org.springframework.cloud.dataflow.server.cloudfoundry.CloudFoundryDataFlowServer</start-class>
+		<reactor.version>2.5.0.M2</reactor.version>
```

Comment:
I prefer the style where all the versions are in the parent pom so it is a one stop shop for changes when making releases.